src/main/java/DataGetSet.java

```diff
@@ -0,0 +1,42 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public  class DataGetSet {
+    public static final String filepath ="C:/Users/kafci/Documents/ip/text-ui-test/savedata.txt";
```

Comment:
File path should be relative to the program location and not global from root.
When others run your program, it will fail because it cannot resolve this file path